dashboard/test/lib/services/user/password_resetter_by_email_test.rb

```diff
@@ -25,6 +25,18 @@ class Services::User::PasswordResetterByEmailTest < ActiveSupport::TestCase
       _(mail.subject).must_equal 'Code.org reset password instructions'
     end
 
+    it 'tracks PasswordResetEmailSuccessful metric' do
+      expected_env = 'expected_env'
+
+      CDO.expects(:rack_env).returns(expected_env)
```

Comment:
Since this part is used in multiple test cases, we can move it to the top of the scope (`describe/context`) for better organization:
```suggestion
      let(:cdo_rack_env) {'expected_cdo_rack_env'}
 
      before do
        CDO.stubs(:rack_env).returns(cdo_rack_env)
      end
```